content/blogs/newsletter-103.md

```diff
@@ -0,0 +1,145 @@
+The answer to these questions is yes -- there will be 0.4.15 and it will be released.
+The 0.4.15 release has been long delayed due to outstanding regressions and bugs.
+
+Most notably it was a regression which made it impossible [to copy files larger than the amount of RAM using Total Commander 9.12](https://jira.reactos.org/browse/CORE-17624), that was long unfixed for some time after the soft rewrite of the Memory Manager.
+While it has been hackfixed by [PR #5630](https://github.com/reactos/reactos/pull/5630), a proper solution is yet to be worked on.
+However, this brings us one step closer to releasing 0.4.15.
+
+Due to development resources and manpower constraints we cannot give an exact estimation when 0.4.15 will be released to the general public but we will eventually get there.
+We are sorry for the inconvenience!
+In the meantime you can try [nightly builds](/getbuilds) that are packed with the latest bugfixes, new features and whatnot but expect also new bugs and such!
+
+## Kernel Debugger overhaul
+Historically, ReactOS was built with the GCC compiler.
+To help with kernel development, a local kernel debugger (KDBG) was developed.
+Unfortunately, KDBG could only understand an in-house debug symbol format that was incompatible with Microsoft's PDB format.
+Because KDBG was a local debugger, it was able to interact with the user via a serial port connection or locally via screen display.
+Input could be entered via serial or a PS/2 keyboard.
+
+Later on, the possibility to compile ReactOS with MSVC was added, and with it, support for the WinDbg debugger.
+Kernel debugging with WinDbg works by having a debugging stub on the target which is capable of manipulating the state of the OS.
+The actual debugger is offloaded to a host computer.
+
+Hermès is updating KDBG with new features and bug fixes to make it easier for developers and testers alike to reliably debug ReactOS.
+Improvements by Hermès for KDBG include:
+* Isolating terminal and input-related functionalities from the rest of KDBG ([#5188](https://github.com/reactos/reactos/pull/5188));
+* Fixed an input buffer reading bug in `KdbpReadCommand()` and `KdReceivePacket()` routines ([#4914](https://github.com/reactos/reactos/pull/4914));
+* Reworked the boot-phase initialization of KD and KDBG ([#4892](https://github.com/reactos/reactos/pull/4892));
+* Moved the code handling of Dmesg buffer from screen provider to KDBG provider ([#5143](https://github.com/reactos/reactos/pull/5143));
+
+The goal of this work is to have a terminal version of KDCOM and separate KDBG into a dedicated KDCOM-like extension DLL, such as `kdrosdbg.dll` (see [#4600](https://github.com/reactos/reactos/pull/4600)).
+All of this effort is further coordinated with Timo Kreuzer.
+
+## UEFI Boot
+Thanks to the work of Timo Kreuzer (tkreuzer) our AMD64 build is now capable of booting on an increasing number of hardware.
```

Comment:
or: "our AMD64 builds" ?

Reply:
Indeed, it's either "AMD64 port" or "AMD64 builds".